### PR TITLE
Data charts

### PR DESCRIPTION
**Pull request by J-Poulter**
# PR Description

Please include a summary of the change:

In this PR, Nick populated the Charts component with the html structure of the page and then the charts themselves which we were then able to get conditionally rendered with values passed in by a dropdown Selection.  Nick also did some styling of the page and the charts, but we decided that the rest of the styling was deserving of its own cards and that we would be adding that task to the project board.

## Type of change
Please uncheck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Add Testing
- [ ] Refactor

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/containers/App/App.css [hunk 1]
```diff
@@ -2,10 +2,22 @@ button:hover {
   cursor: pointer;
 }
 
+.app-container {
+
+  height: auto;
+  /* min-height: 90vh;  */
+  width: 100%;
+
+}
```
Comment (nickstaylor) [inline src/containers/App/App.css:11]: We need to clean up our styling here to make all of the components uniform

File: src/containers/Charts/Chart.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      backgroundColor: 'rgba(75,192,192,1)',
+      borderColor: 'whitesmoke',
+      borderWidth: 2,
+      data: solarData.solrad_monthly,
+
+    }]
+  })
+
+  const getSpecificChart = (e) => {
+    let chartName = e.target.options[e.target.selectedIndex].text
```
Comment (nickstaylor) [inline src/containers/Charts/Chart.js:27]: Great work on this dynamic method for being able to toggle between data sets and graphs

File: src/containers/Charts/Chart.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+import './Chart.css'
+
+
+const Chart = ({ solarData }) => {
+
+  const [chartType, updateChartType] = useState('bar')
+  const [yAxisLabel, updateYAxisLabel] = useState('(kWh / m^2) / day')
```
Comment (alex-latham) [inline src/containers/Charts/Chart.js:12]: Smart use of state to hold the axis labels

File: src/containers/Configure/Configure.js [hunk 4]
```diff
@@ -149,7 +149,9 @@ const mapStateToProps = () => ({
 
 const mapDispatchToProps = (dispatch) => ({
   userQuote: (quote) => dispatch(actions.userQuote(quote)),
-  allUserQuotes: (userQuote) => dispatch(actions.allUserQuotes(userQuote))
+  allUserQuotes: (userQuote) => dispatch(actions.allUserQuotes(userQuote)),
+  loadSolarData: (data) => dispatch(actions.loadSolarData(data))
```
Comment (alex-latham) [inline src/containers/Configure/Configure.js:153]: Good catch adding the config data here.

**Pull request merged**